### PR TITLE
First phase of #1

### PR DESCRIPTION
**Pull request by patriciakasapa**
My first push to the TCMS project.

### CODE REVIEW

File: src/main/java/io/turntabl/TCMS.java [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+    public String toString() {
+        return "TCMS{}";
+    }
+
     public static void main(String[] args) {
+
+        String pattern;
+        String check = null;
+
+        Scanner input = new Scanner(System.in);
```
Comment (idawud) [inline src/main/java/io/turntabl/TCMS.java]: Can you move all this codes to a dedicate get userInputs() func ?

File: src/main/java/io/turntabl/TCMS.java [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+        System.out.println("Address: ");
+        String address = input.nextLine();
+
+        System.out.println("Telephone number: ");
+        String tel = input.nextLine();
+
+        System.out.println("Email Address: ");
+        String email = input.nextLine();
+
+        System.out.println("Name: " + "  " + name + "  " +"Address: " + "  " + address + "  " + "Telephone: " + "  " + tel + "  " + "Email: " + "  " + email);
```
Comment (pkErbynn) [inline src/main/java/io/turntabl/TCMS.java]: maybe you can end the lines

File: src/main/java/io/turntabl/entering_details/Client_info_entry.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    String email;
+
+    public Client_info_entry(String name, String address, String tel, String email) {
+        this.name = name;
+        this.address = address;
+        this.tel = tel;
+        this.email = email;
+    }
+
+    public Client_info_entry(Scanner input) {
```
Comment (pkErbynn) [inline src/main/java/io/turntabl/entering_details/Client_info_entry.java]: ...this fxn seem to be idle

File: src/main/java/io/turntabl/entering_details/Client_info_entry.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package io.turntabl.entering_details;
+
+import java.util.Scanner;
+
+public class Client_info_entry {
```
Comment (idawud) [inline src/main/java/io/turntabl/entering_details/Client_info_entry.java]: Looks good but I also have one, so we could use my when integrating

File: src/test/java/io/turntabl/entering_details/Client_info_entryTest.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package io.turntabl.entering_details;
+
+import org.junit.Test;
+
+public class Client_info_entryTest {
+
+
```
Comment (pkErbynn) [inline src/test/java/io/turntabl/entering_details/Client_info_entryTest.java]: ....code spacing consistency....maybe you can leave just a line

File: src/test/java/io/turntabl/entering_details/Client_info_entryTest.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package io.turntabl.entering_details;
```
Comment (idawud) [inline src/test/java/io/turntabl/entering_details/Client_info_entryTest.java]: good! move the inputs from main to this function

**Review by idawud — APPROVED** [review]

**Review by pkErbynn — COMMENTED** [review]
Nice work done

**Pull request merged**